web3/_utils/module_testing/eth_module.py

```diff
@@ -1454,6 +1454,54 @@ async def test_eth_getBlockTransactionCountByNumber_block_with_txn(
         assert is_integer(transaction_count)
         assert transaction_count >= 1
 
+    async def test_async_eth_sign(
```

Comment:
Ah, sorry, I found out that `test_async_eth_sign` is missing `@pytest.mark.asyncio`, I didn't check its CI warnings.